src/pages/EnvironmentsPage.ts

```diff
@@ -1,8 +1,14 @@
 import { NavLinkProvider } from '@terra-ui-packages/core-utils';
+import { ReactNode } from 'react';
 import { h } from 'react-hyperscript-helpers';
-import { EnvironmentNavActions, Environments } from 'src/analysis/Environments/Environments';
+import { EnvironmentNavActions, Environments, UseWorkspacesState } from 'src/analysis/Environments/Environments';
```

Comment:
Shouldn't `UseWorkspaceState` be a shared interface similar to various providers?

Reply:
yes, but didn't want to take on converting the workspace utils file it currently comes from to TS since this PR was already on the big side.  But I can do that as a small followup....